### PR TITLE
Update CONTRIBUTING. including doc

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Update CONTRIBUTING. including doc 
# Checklist

- [ ] Doxygen documentation of the code completed (classes, methods, types, members...)
- [ ] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [ ] Update the readme with potentially a screenshot of the tools if it applies. 
- [ ] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -104,7 +104,13 @@ scripts. This script generates a new empty tool containing only these minimal fe
                  << "3dCurveViewer -C -b 1 -3 -2 -c ${DGtal}/examples/samples/sinus.dat\n";
       return 0;
 ```
-
+  - doxygen tool documentation: you have to add the documentation in the doxygen format including the following elements:
+    - brief description of the tool,
+    - main usage,
+    - list of the options,
```
Comment (dcoeurjo) [inline CONTRIBUTING.md]: options -> parameters

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -104,7 +104,13 @@ scripts. This script generates a new empty tool containing only these minimal fe
...
+    - typical example of use with result illustration (if available)
```
Comment (dcoeurjo) [inline CONTRIBUTING.md]: typical examples with illustrations (if available) ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -104,7 +104,13 @@ scripts. This script generates a new empty tool containing only these minimal fe
...
+   
+   You can use as model the documentation of one tool of the main DGtalTools [documentation](http://dgtal.org/doc/tools/nightly/) and start from the template generated by the ```newtools.sh```script.
```
Comment (dcoeurjo) [inline CONTRIBUTING.md]: DGtalScripts is not really well exposed.. maybe nobody will understand what you mean by "newtools.sh"

File: CONTRIBUTING.md (hunk 2 above)
Comment (kerautret) [inline CONTRIBUTING.md]: yes thanks

File: CONTRIBUTING.md (hunk 3 above)
Comment (kerautret) [inline CONTRIBUTING.md]: you are right I add a link to DGtalScripts ;)

**Comment by kerautret** [comment]
@dcoeurjo thanks done

**Comment by kerautret** [comment]
can we merge ? ping @dcoeurjo  ;)

**Comment by dcoeurjo** [comment]
Yep, merging

**Pull request merged**